FreeRDP/FreeRDP
# assistance v2 support

Fixes #4987 

* Adds parsing support for newer encrypted `msrcIncident` files.
* Parsing and settings initialization now unified in `assistance` file module

Testing:
1. goto `System Properties/Remote Tab`
1. Check `Allow Remote Assistance connections to this computer`
1. Click the `Advanced...` Button
1. Check `Create invitations that can only be used from computers running Windows Vista or later`

Run:
1. Start `msra`
1. Click `Invite someone you trust to help you`
1. Click `Save this invitation as a file`
1. Remember the password and copy the file to your client test machine
1. Connect using `xfreerpd <file>.msrcIncident /assistance:<password>`, add `/cert-ignore` if the certificate is not already trusted.

1. Uncheck `Create invitations that can only be used from computers running Windows Vista or later` and repeat this test
